tcms/testplans/static/testplans/js/get.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+    const testPlanId = $('#test_plan_pk').data('testplanPk');
+    const testCases = {
+        'confirmed': [],
+        'not_confirmed': []
+    };
+
+    jsonRPC('TestCase.filter', [{'plan': testPlanId}], function(data) {
+        for (var i = 0; i < data.length; i++) {
+            var testCase = data[i];
+            if (testCase.case_status_id === 2) {
+                testCases.confirmed.push(data[i]);
+            } else {
+                testCases.not_confirmed.push(data[i]);
+            }
+        }
+        drawConfirmedTestCases();
+
+    });
+
+    function drawConfirmedTestCases() {
+        var container = $('#confirmed-testcases'),
+            noCasesTemplate = $('#no_test_cases'),
+            testCaseRow = $('#test_case_row');
+
+        if (testCases.confirmed.length > 0) {
+            for (var i = 0; i < testCases.confirmed.length; i++) {
+                var testCase = testCases.confirmed[i];
+                testCaseRow[0].content.querySelector('.list-view-pf-main-info').innerText = `TC-${testCase.id}: ${testCase.summary}`;
```

Comment:
Looks like indexing operation can be moved above when initializing the variable. I don't see why it is necessary to do it on every loop iteration. 

Also `.content.querySelector` can be replaced with `$(testCaseRow).find()` I think. We've got jQuery at our disposal, use of it if it makes the syntax shorter/easier to read.